src/type_set.rs

```diff
@@ -1,17 +1,20 @@
 //! Type-level set inclusion and difference, inspired by frunk's approach: <https://archive.is/YwDMX>
+#[cfg(feature = "error_provide")]
+use crate::maybe_std::error;
+use crate::maybe_std::error::Error;
+use crate::maybe_std::Box;
 use core::any::Any;
 use core::fmt;
-use std::error::Error;
 
 use crate::{Cons, End, Recurse};
 
-/* ------------------------- std::error::Error support ----------------------- */
+/* ------------------------- Error support ----------------------- */
 
 pub trait ErrorFold {
     fn source_fold(any: &Box<dyn Any>) -> Option<&(dyn Error + 'static)>;
 
     #[cfg(feature = "error_provide")]
-    fn provide_fold<'a>(any: &'a Box<dyn Any>, request: &mut std::error::Request<'a>);
+    fn provide_fold<'a>(any: &'a Box<dyn Any>, request: &mut error::Request<'a>);
```

Comment:
```suggestion
    fn provide_fold<'a>(any: &'a Box<dyn Any>, request: &mut core::error::Request<'a>);
```
Using a full path here means less lines of code are covered by feature flags, making it easier to reason about whether setting the flag compiles.